website/docs/product/telemetry.md

```diff
@@ -91,52 +139,72 @@ The client captures anonymous behavioral data around navigation and clicks.
+### Successful backend calls
+The server shares anonymous information about successful query processes, new application creation, user logins, connections to additional plugins, etc. This data is only collected when telemetry is turned on. 
 
-```javascript
+```javascript title="sample successful backend call"
 {
-  "usageData": [
-    {
-      "user": "f3273dd18d95bc19d51d3e6356e4a679e6f13824497272a270e7bb540b0abb9d", // unique hashed-id for each user
-      "tenantId": "6fh76357fbe7e44f3a47a", 
-      // flag to determine if usage occurred in view mode or edit mode of application. We don’t charge developers when they are building their apps in edit mode
-      "viewMode": false, 
-      "isAnonymousUser": false, // flag to determine if pulse was triggered by logged-in user or an anonymous user
-      "createdAt": 188474747 // timestamp of usage pulse
-    }
-  ],
-  //message and hashMessage are for cloud-services to check the integrity of the data, ensuring that the pulses are sent by the real instance owning this license key
-  "message": "hash-message",
-  "hashedMessage": "c8ec6166d030765ff0f88ce40f4494bc6ef99f9d65dfbecd974c6359d1cac7ac",
-  "instanceId": "63ef757fbe7e44f3a47a"
+    "context": {
+        "library": {
+            "name": "analytics-java",
+            "version": "2.1.1"
+        }
+    },
+    "event": "execute_ACTION_TRIGGERED",
+    "integrations": {},
+    "messageId": "0f6b07ee-0717-413-808c-c25b09c0468",
+    "originalTimestamp": "2021-08-24T07:23:35.610Z",
+    "properties": {
+        "appId": "612465f87b2230debedfc6",
+        "appMode": "edit",
+        "appName": "APP1",
+        "datasource": {
+            "name": "Test App"
+        },
+        "instanceId": "612460418944011a10fa5b",
+        "isExampleApp": false,
+        "isSuccessfulExecution": true,
+        "name": "Test",
+        "orgId": "612464f7f230debedfc4",
+        "originService": "appsmith-server",
+        "pageId": "612465802230debedfc8",
+        "pageName": "Page1",
+        "pluginName": "PostgreSQL",
+        "statusCode": "",
+        "timeElapsed": 8,
+        "type": "DB",
+        "username": "70280e5d07e61e5e915e5d26ac8704bbd68d3f75ebad67ba439f4c354d7"
+    },
+    "receivedAt": "2021-08-24T07:23:39.996Z",
+    "sentAt": "2021-08-24T07:23:39.885Z",
+    "timestamp": "2021-08-24T07:23:35.721Z",
+    "type": "track",
+    "userId": "70280e5dd9e61e5e91526ac8704bbd68d3f75ebad67ba439f4c354d7",
 }
-
 ```
 
 ## Disable telemetry
 
-You can either disable the telemetry from [**Admin Settings**](telemetry.md#admin-settings) or by making changes to the [**`docker.env`**](telemetry.md#docker.env-file) file.
+Telemetry sharing is an optional setting, and you can disable telemetry either from Admin Settings or by making changes to the environment variable.
```

Comment:
Can we just say "Sharing telemetry is optional, and you can disable telemetry..."?